driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -64,7 +64,111 @@ public static Bson addFields(final Field<?>... fields) {
      * @since 3.4
      */
     public static Bson addFields(final List<Field<?>> fields) {
-        return new AddFieldsStage(fields);
+        return new FieldsStage("$addFields", fields);
+    }
+
+    /**
+     * Creates an $accumulator pipeline stage
```

Comment:
`$accumulator` is not a pipeline stage (see https://docs.mongodb.com/master/reference/operator/aggregation-pipeline/) , it's an accumulator (see https://docs.mongodb.com/master/reference/operator/aggregation/group/).  Hence, it seems more appropriate that these methods would go in the `com.mongodb.client.model.Accumulators` class along with the rest of the accumulator helpers.

Reply:
I have moved these methods to `Accumulators`.